NetKAN/TweakScaleRescaled-Redist.netkan

```diff
@@ -0,0 +1,26 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled-Redist
+name: TweakScale Rescaled Redistributable
+abstract: >-
+  Provides a mechanism for other mods to interact with TweakScaleRescaled.
+  This plugin has no overhead when TweakScaleRescaled is not installed.
+$kref: '#/ckan/spacedock/3514'
```

Comment:
Applicable to all three modules: Any objections to me adding support for hosting on both GitHub and SpaceDock? This should increase reliability and probably download speeds, since so many people apparently have slow routes to SD.

Reply:
I like multi-hosting in theory, I don't like the added complexity in the netkan file (we had a short conversation about this).  If it doesn't get *too* crazy I'm fine with it, but maybe let's get this in first and then add multi-hosting later?